principles.qmd

```diff
@@ -119,6 +119,15 @@ Consistent, high quality, and accessible documentation is key to adopting softwa
 - Aim to provide translations for the most common languages
 - We aim to provide [automated blog posts for major and minor releases](https://epiverse-trace.github.io/blog.html#category=new-release). Code owners may occasionally decide to release patch blogs if the patch is critical.
 
+### Website modes
+
+Epiverse-TRACE packages use [`pkgdown`](https://pkgdown.r-lib.org/) to generate websites. `pkgdown` provides [options](https://pkgdown.r-lib.org/reference/build_site.html#development-mode) (in the `_pkgdown.yml` file) to generate different sites for the development and released versions of your package.
+
+When the package is not yet on CRAN, we use the `unreleased` mode. After the package is on CRAN, we switch to `auto`, which provides two versions of the website: `release` and `devel`. This is because we expect most users to install from CRAN and so we direct them to the frozen release site, with an option to manually switch to the `devel` website if they installed from GitHub.
+
+We also provide text in the README to direct users to the appropriate site. Typical text for the README might be: "The package has two websites: one for `[the stable release version on CRAN](URL)`, and
```

Comment:
Has this been discussed and agreed? I can add this to the packages I maintain if so.

Reply:
> Has this been discussed and agreed? I can add this to the packages I maintain if so.

It was discussed in the issue comments and I am putting it here prospectively. We can agree to keep or remove it.